docs/source/quickstart.rst

```diff
@@ -91,187 +113,206 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
+- For more details on the explicit logging API, see :ref:`tracking_logging_functions`.
 
-        # Log a parameter (key-value pair)
-        mlflow_log_param("param1", 5)
-
-        # Log a metric; metrics can be updated throughout the run
-        mlflow_log_metric("foo", 1)
-        mlflow_log_metric("foo", 2)
-        mlflow_log_metric("foo", 3)
-
-        # Log an artifact (output file)
-        writeLines("Hello world!", "output.txt")
-        mlflow_log_artifact("output.txt")
-
-Viewing the Tracking UI
------------------------
+View MLflow runs and experiments
+-----------------------------------
 
-By default, wherever you run your program, the tracking API writes data into files into a local
-``./mlruns`` directory. You can then run MLflow's Tracking UI:
+Once you've run your code, you may view the results with MLflow's tracking UI. To start the UI, run:
 
 .. code-section::
 
     .. code-block:: shell
 
         mlflow ui
 
-    .. code-block:: R
+And then navigate to http://localhost:5000 in your browser. You will see a page similar to:
 
-        mlflow_ui()
+.. image:: _static/images/quickstart/quickstart_ui_screenshot.png
+    :width: 800px
+    :align: center
+    :alt: Screenshot of MLflow tracking UI
+..
 
-and view it at http://localhost:5000.
+You are in the **Default** experiment, which now contains the tracking data for your run. An experiment is a collection of related runs. You can view the parameters, metrics, and artifacts of your run by clicking on the run name.
 
-.. note::
-    If you see message ``[CRITICAL] WORKER TIMEOUT`` in the MLflow UI or error logs, try using ``http://localhost:5000`` instead of ``http://127.0.0.1:5000``.
+For more details on the tracking UI, see :ref:`tracking`.
 
+.. _quickstart_tracking_server:
 
-Running MLflow Projects
------------------------
+Share MLflow runs and experiments
+-----------------------------------
 
-MLflow allows you to package code and its dependencies as a *project* that can be run in a
-reproducible fashion on other data. Each project includes its code and a ``MLproject`` file that
-defines its dependencies (for example, Python environment) as well as what commands can be run into the
-project and what arguments they take.
+For getting started, the last example stored the tracking data locally. Generally, you will want to use shared storage. Locally, MLflow stores tracking data and artifacts in an **mlruns/** subdirectory of where you ran the code. The tracking UI, when run locally, visualizes this. You can store your data remotely by:
 
-You can easily run existing projects with the ``mlflow run`` command, which runs a project from
-either a local directory or a GitHub URI:
+- Calling ``mlflow.set_tracking_uri`` in your code; or
+- Setting the ``MLFLOW_TRACKING_URI`` environment variable
 
-.. code-block:: bash
+With either option, you can track your runs in a shared filesystem, a SQLAlchemy-compatible database, a tracking server, or a Databricks workspace.
```

Comment:
Love that we're calling this out! I found it a bit confusing that we're telling people to call this Python API / set this environment variable before we tell them about the possible values they can use (file system URI for shared filesystem, SQLAlchemy URI for database, HTTP/S URL for tracking server, etc).

Perhaps we should move this below the discussion of "what is a tracking server?". It would also be great to include a couple of examples showing how you can use `set_tracking_uri` and `MLFLOW_TRACKING_URI` with a shared filesystem and with a SQL database. Can we also link to the relevant MLflow Tracking docs so that users can find more info?